test/features/groups.feature

```diff
@@ -1,53 +1,34 @@
 @users
 Feature: Group APIs
 
-    Scenario: Group membership is accessible to sysadmins
-        Given "SysAdmin" as the persona
+    Scenario Outline: Group membership is accessible to admins of the group
+        Given "<User>" as the persona
         When I log in
         And I view the "silly-walks" group API "including" users
         Then I should see an element with xpath "//*[contains(string(), '"success": true,') and contains(string(), '"name": "group_admin"') and contains(string(), '"name": "walker"')]"
 
-    Scenario: Group membership is accessible to admins of the group
-        Given "Group Admin" as the persona
-        When I log in
-        And I view the "silly-walks" group API "including" users
-        Then I should see an element with xpath "//*[contains(string(), '"success": true,') and contains(string(), '"name": "group_admin"') and contains(string(), '"name": "walker"')]"
+        Examples: Admins
+            | User        |
```

Comment:
is user an admin?

Reply:
The first 'example' is the field name to use.

Reply:
ah, i see. tho may be better to call it 'admins' instead of user since don't we also have a 'user' persona?

Reply:
How about 'Account'?